package.json

```
@@ -1,8 +1,12 @@
 {
     "name": "filament",
     "scripts": {
-        "build": "node bin/build && mix --production",
-        "watch": "node bin/build && mix watch",
+        "dev:scripts": "node bin/build --watch",
+        "dev:styles": "mix watch",
+        "build:scripts": "node bin/build",
+        "build:styles": "mix --production",
+        "watch": "npm-run-all --parallel dev:*",
```

Rename this to `dev` for consistency?
```suggestion
        "dev": "npm-run-all --parallel dev:*",
```